adding the coastal ghb to cells with land surface or river stage belo…

janetrbarclay opened the pull request:
…w sea level (works best with a combined dem / bathymetry layer)


Parameters and approach are based on  Befus, Kevin M., Kevin D. Kroeger, Christopher G. Smith, and Peter W. Swarzenski. “The Magnitude and Origin of Groundwater Discharge to Eastern U.S. and Gulf of Mexico Coastal Waters.” Geophysical Research Letters 44, no. 20 (October 28, 2017): 10,396-10,406. https://doi.org/10.1002/2017GL075238.


**gen_mod_dict.py:** 
I added a scenario and a few parameters (sea level, salt water denisty, freshwater density, sediment thickness and vertical anisotropy) to gen_mod_dict. 

**Notebook 1:**
The depth below sea level is calculated for each cell with a river stage or land surface below sea level. Then the salt water head is converted to an equivalent freshwater head and save in model_grid. 

**Notebook 3:**
Conductance for the ghb is calculated using the freshwater head from notebook 1 and the sediment parameters from gen_mod_dict

janetrbarclay committed:
adding the coastal ghb to cells with land surface or river stage below sea level (works best with a combined dem / bathymetry layer)

jjstarn-usgs commented:
thanks!  looks good; more through than what we had planned


Jeff Starn Ph.D.
Research Hydrologist
USGS National Water-Quality Assessment
101 Pitkin Street,  East Hartford, CT 06108

jjstarn@usgs.gov
office: (860) 291-6746
https://www.usgs.gov/staff-profiles/jon-jeffrey-starn



On Mon, Mar 11, 2019 at 11:39 AM Janet Barclay <notifications@github.com>
wrote:

> …w sea level (works best with a combined dem / bathymetry layer)
>
> Parameters and approach are based on Befus, Kevin M., Kevin D. Kroeger,
> Christopher G. Smith, and Peter W. Swarzenski. “The Magnitude and Origin of
> Groundwater Discharge to Eastern U.S. and Gulf of Mexico Coastal Waters.”
> Geophysical Research Letters 44, no. 20 (October 28, 2017): 10,396-10,406.
> https://doi.org/10.1002/2017GL075238.
>
> *gen_mod_dict.py:*
> I added a scenario and a few parameters (sea level, salt water denisty,
> freshwater density, sediment thickness and vertical anisotropy) to
> gen_mod_dict.
>
> *Notebook 1:*
> The depth below sea level is calculated for each cell with a river stage
> or land surface below sea level. Then the salt water head is converted to
> an equivalent freshwater head and save in model_grid.
>
> *Notebook 3:*
> Conductance for the ghb is calculated using the freshwater head from
> notebook 1 and the sediment parameters from gen_mod_dict
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/usgs/gw-general-models/pull/3
> Commit Summary
>
>    - adding the coastal ghb to cells with land surface or river stage
>    below sea level (works best with a combined dem / bathymetry layer)
>
> File Changes
>
>    - *M*
>    Genmod1.0/general-models/UserGuide_General_Model_1_Geospatial_processing_v2.0_GESS.ipynb
>    <https://github.com/usgs/gw-general-models/pull/3/files#diff-0> (57)
>    - *M*
>    Genmod1.0/general-models/UserGuide_General_Model_3_Create_the_Model--Multi_layer_GESS.ipynb
>    <https://github.com/usgs/gw-general-models/pull/3/files#diff-1> (43)
>    - *M* Genmod1.0/general-models/gen_mod_dict.py
>    <https://github.com/usgs/gw-general-models/pull/3/files#diff-2> (23)
>
> Patch Links:
>
>    - https://github.com/usgs/gw-general-models/pull/3.patch
>    - https://github.com/usgs/gw-general-models/pull/3.diff
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/usgs/gw-general-models/pull/3>, or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGRb8VO3Pee0pCm7fqsfaO_bWzL1Yhkhks5vVniYgaJpZM4bo6WV>
> .
>

janetrbarclay commented:
Thanks! I mostly just used the approach from that paper and implemented it
using the framework you all had set up.

Janet

On Mon, Mar 11, 2019 at 3:03 PM Jeff Starn <notifications@github.com> wrote:

> thanks! looks good; more through than what we had planned
>
>
> Jeff Starn Ph.D.
> Research Hydrologist
> USGS National Water-Quality Assessment
> 101 Pitkin Street, East Hartford, CT 06108
>
> jjstarn@usgs.gov
> office: (860) 291-6746
> https://www.usgs.gov/staff-profiles/jon-jeffrey-starn
>
>
>
> On Mon, Mar 11, 2019 at 11:39 AM Janet Barclay <notifications@github.com>
> wrote:
>
> > …w sea level (works best with a combined dem / bathymetry layer)
> >
> > Parameters and approach are based on Befus, Kevin M., Kevin D. Kroeger,
> > Christopher G. Smith, and Peter W. Swarzenski. “The Magnitude and Origin
> of
> > Groundwater Discharge to Eastern U.S. and Gulf of Mexico Coastal Waters.”
> > Geophysical Research Letters 44, no. 20 (October 28, 2017):
> 10,396-10,406.
> > https://doi.org/10.1002/2017GL075238.
> >
> > *gen_mod_dict.py:*
> > I added a scenario and a few parameters (sea level, salt water denisty,
> > freshwater density, sediment thickness and vertical anisotropy) to
> > gen_mod_dict.
> >
> > *Notebook 1:*
> > The depth below sea level is calculated for each cell with a river stage
> > or land surface below sea level. Then the salt water head is converted to
> > an equivalent freshwater head and save in model_grid.
> >
> > *Notebook 3:*
> > Conductance for the ghb is calculated using the freshwater head from
> > notebook 1 and the sediment parameters from gen_mod_dict
> > ------------------------------
> > You can view, comment on, or merge this pull request online at:
> >
> > https://github.com/usgs/gw-general-models/pull/3
> > Commit Summary
> >
> > - adding the coastal ghb to cells with land surface or river stage
> > below sea level (works best with a combined dem / bathymetry layer)
> >
> > File Changes
> >
> > - *M*
> >
> Genmod1.0/general-models/UserGuide_General_Model_1_Geospatial_processing_v2.0_GESS.ipynb
> > <https://github.com/usgs/gw-general-models/pull/3/files#diff-0> (57)
> > - *M*
> >
> Genmod1.0/general-models/UserGuide_General_Model_3_Create_the_Model--Multi_layer_GESS.ipynb
> > <https://github.com/usgs/gw-general-models/pull/3/files#diff-1> (43)
> > - *M* Genmod1.0/general-models/gen_mod_dict.py
> > <https://github.com/usgs/gw-general-models/pull/3/files#diff-2> (23)
> >
> > Patch Links:
> >
> > - https://github.com/usgs/gw-general-models/pull/3.patch
> > - https://github.com/usgs/gw-general-models/pull/3.diff
> >
> > —
> > You are receiving this because you are subscribed to this thread.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/usgs/gw-general-models/pull/3>, or mute the thread
> > <
> https://github.com/notifications/unsubscribe-auth/AGRb8VO3Pee0pCm7fqsfaO_bWzL1Yhkhks5vVniYgaJpZM4bo6WV
> >
> > .
> >
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/usgs/gw-general-models/pull/3#issuecomment-471677772>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADp_0MrPVactt7OZDWVDK59z-c0_-hhvks5vVqiMgaJpZM4bo6WV>
> .
>
